### DISCUSSION TITLE
How to do a multi-instrument crosssectional long-short strategy

### DISCUSSION THREAD

**Discussion by rrrwa** [nautechsystems/nautilus_trader#1946] [Q&A] [answered]
Hello, I'm a new user of this system. I've just gone through the examples strategies and some component source code, but all the strategies only involve a single instrument. I hope to implement those strategies that involve multiple instruments at the same time and do a long short relative strength trading on them. Thanks

**Comment by cjdsellers** [comment, marked as answer]
Hi @rrrwa

Thanks for your interest!

There's no limitation on number of instruments per strategy, it just comes down to writing the Python to add them through your own strategy configuration object or fetching from the `self.cache`. We'll be sure to add multi-instrument examples in the future.
